flink-table/flink-sql-parser/src/main/codegen/data/Parser.tdd

```diff
@@ -21,69 +21,70 @@
-    "org.apache.flink.sql.parser.ddl.SqlAlterFunction",
-    "org.apache.flink.sql.parser.ddl.SqlCreateFunction",
-    "org.apache.flink.sql.parser.ddl.SqlDropFunction",
-    "org.apache.flink.sql.parser.ddl.SqlAlterTable",
-    "org.apache.flink.sql.parser.ddl.SqlAlterTableRename",
-    "org.apache.flink.sql.parser.ddl.SqlAlterTableProperties",
-    "org.apache.flink.sql.parser.ddl.SqlCreateCatalog",
-    "org.apache.flink.sql.parser.dml.RichSqlInsert",
-    "org.apache.flink.sql.parser.dml.RichSqlInsertKeyword",
-    "org.apache.flink.sql.parser.dql.SqlShowCatalogs",
-    "org.apache.flink.sql.parser.dql.SqlDescribeCatalog",
-    "org.apache.flink.sql.parser.dql.SqlShowDatabases",
-    "org.apache.flink.sql.parser.dql.SqlShowFunctions",
-    "org.apache.flink.sql.parser.dql.SqlDescribeDatabase",
-    "org.apache.flink.sql.parser.dql.SqlShowTables",
-    "org.apache.flink.sql.parser.dql.SqlRichDescribeTable",
-    "org.apache.flink.sql.parser.type.ExtendedSqlBasicTypeNameSpec",
-    "org.apache.flink.sql.parser.type.ExtendedSqlCollectionTypeNameSpec",
-    "org.apache.flink.sql.parser.utils.SqlTimeUnit",
-    "org.apache.flink.sql.parser.utils.ParserResource",
-    "org.apache.flink.sql.parser.validate.FlinkSqlConformance",
-    "org.apache.flink.sql.parser.SqlProperty",
-    "org.apache.calcite.sql.SqlDrop",
-    "org.apache.calcite.sql.SqlCreate",
-    "java.util.List",
+    "org.apache.flink.sql.parser.ddl.SqlDropView"
+    "org.apache.flink.sql.parser.ddl.SqlTableColumn"
+    "org.apache.flink.sql.parser.ddl.SqlTableOption"
+    "org.apache.flink.sql.parser.ddl.SqlUseCatalog"
+    "org.apache.flink.sql.parser.ddl.SqlUseDatabase"
+    "org.apache.flink.sql.parser.ddl.SqlWatermark"
+    "org.apache.flink.sql.parser.dml.RichSqlInsert"
+    "org.apache.flink.sql.parser.dml.RichSqlInsertKeyword"
+    "org.apache.flink.sql.parser.dql.SqlDescribeCatalog"
+    "org.apache.flink.sql.parser.dql.SqlDescribeDatabase"
+    "org.apache.flink.sql.parser.dql.SqlShowCatalogs"
+    "org.apache.flink.sql.parser.dql.SqlShowDatabases"
+    "org.apache.flink.sql.parser.dql.SqlShowFunctions"
+    "org.apache.flink.sql.parser.dql.SqlShowTables"
+    "org.apache.flink.sql.parser.dql.SqlRichDescribeTable"
+    "org.apache.flink.sql.parser.type.ExtendedSqlCollectionTypeNameSpec"
+    "org.apache.flink.sql.parser.type.ExtendedSqlRowTypeNameSpec"
+    "org.apache.flink.sql.parser.type.SqlMapTypeNameSpec"
+    "org.apache.flink.sql.parser.utils.SqlTimeUnit"
+    "org.apache.flink.sql.parser.utils.ParserResource"
+    "org.apache.flink.sql.parser.validate.FlinkSqlConformance"
+    "org.apache.flink.sql.parser.SqlProperty"
+    "org.apache.calcite.sql.SqlAlienSystemTypeNameSpec"
+    "org.apache.calcite.sql.SqlCreate"
+    "org.apache.calcite.sql.SqlDrop"
+    "java.util.List"
     "java.util.ArrayList"
   ]
 
   # List of new keywords. Example: "DATABASES", "TABLES". If the keyword is not a reserved
   # keyword, please also add it to 'nonReservedKeywords' section.
```

Comment:
add  "Please keep the keywords in alphabetical order if new keyword is added." here?

Reply:
Agree, thanks ~